Makefile

```diff
@@ -116,7 +116,7 @@ e2e: e2e-image
 e2e: $(GINKGO) $(KUSTOMIZE) $(KIND) $(GOVC) ## Run e2e tests
 	$(MAKE) release-manifests
 	@mkdir -p $(E2E_TEMPLATE_DIR)
-	cp $(RELEASE_DIR)/cluster-template.yaml $(E2E_TEMPLATE_DIR)
+	cp $(RELEASE_DIR)/cluster-template-haproxy.yaml $(E2E_TEMPLATE_DIR)
```

Comment:
When would you want to switch e2e to use kube-vip?

Reply:
Ideally once DHCP is supported in kube-vip, or when doing some testing around VIPs from other segments in our infra.